test/json-ld.net.tests/Sorting/W3C/compact-in.jsonld

```diff
@@ -0,0 +1,89 @@
+﻿[
```

Comment:
After getting `FromRDF` to work, I tried the same approach with `Compact`, but it seemed too complicated to get it to work. Plus, if `Compact` or the other methods are given input, they seem to maintain the order. I left the remnants of the test here though in case it seemed worthwhile to try again later.

Or we can just delete it.